tools/ci/src/commands/compile_check_no_std.rs

```diff
@@ -0,0 +1,35 @@
+use crate::{Flag, Prepare, PreparedCommand};
+use argh::FromArgs;
+use xshell::cmd;
+
+/// Checks that the project compiles for a `no_std` target.
+#[derive(FromArgs, Default)]
+#[argh(subcommand, name = "compile-check-no-std")]
+pub struct CompileCheckNoStdCommand {}
```

Comment:
could you make the target as a parameter to the subcommand, defaulting to `x86_64-unknown-none`?

Reply:
Can do! First time working with `argh` but I think I've done this correctly.